api/src/test/java/net/kyori/adventure/bossbar/BossBarTest.java

```diff
@@ -25,17 +25,24 @@
 
 import com.google.common.collect.ImmutableSet;
 import java.util.Collections;
+import java.util.Locale;
 import java.util.Set;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicReference;
+
```

Comment:
Just gonna bug you about this before Kash does ;)

Reply:
What exactly is the issue here? Also, blame IntelliJ for that

Reply:
Imports need to be all together in alphabetical order and the only newline is between normal and static imports

IDEA does love messing up imports dw :')